views/default/page/layouts/widgets/add_button.php

```diff
@@ -17,8 +17,8 @@
 		[
 			'name' => 'widgets_add',
 			'href' => 'javascript: void(0)',
-			'text' => elgg_echo('widgets:add'),
-			'link_class' => 'elgg-button elgg-button-action elgg-lightbox',
+			'text' => elgg_view_icon('cog') . ' ' . elgg_echo('widgets:add'),
```

Comment:
as menu items use output/url you can find docs in that view about 'icon' and 'badge'